src/application-events-model.ts

```diff
@@ -590,12 +590,12 @@ export const ApplicationEventsModel: { [playerEventName: string]: KavaEvent } =
       eventType: ApplicationEventType.BUTTON_CLICKED,
       eventVar1: 'Advanced_captions_font_opacity',
       eventVar2: ButtonType.Choose,
-      eventVar3: `${String(payload).split('.')[1]}%`
+      eventVar3: `${payload * 100}%`
```

Comment:
is payload a float or an integer ?

Reply:
Float

Reply:
I mean, do you need to round up the number or will it always be rounded ?

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/opacity
it could be Any [<number>](https://developer.mozilla.org/en-US/docs/Web/CSS/number) strictly between 0 and 1

and I just need to convert it to a percentage 

0.434 * 100 => 43.4
0.44 * 100 => 43